.github/workflows/update-server-side-sdk-schema.yml

```diff
@@ -108,23 +123,32 @@ jobs:
         env:
           CI: true
 
-      - name: 'Create branch'
-        run: |
-          git checkout -b feat/open-api-${{ inputs.tag }}
-
       - name: 'Install dependencies'
         run: pnpm install
         working-directory: dx-team-toolkit
 
+      - name: Set git config
+        run: |
+          git config --global --add --bool push.autoSetupRemote true
+
       - name: Update schema
-        run: node dx-team-toolkit/scripts/update-schema.mjs ${{ inputs.schema-path }} ${{ inputs.examples-path }} ${{ inputs.generate-command }} ${{ inputs.tag }} ${{ secrets.GITHUB_TOKEN }}
+        run: node dx-team-toolkit/scripts/update-schema/update-schema.mjs "${{ inputs.schema-path }}" "${{ inputs.examples-path }}" "${{ inputs.generate-command }}" ${{ inputs.tag }} ${{ inputs.appId != '' && steps.app-token.outputs.token || secrets.GH_TOKEN }}
 
-      - name: Push changes
-        run: |
-          git push origin feat/open-api-${{ inputs.tag }}
+      - name: Remove DX toolkit directory
+        run: rm -rf dx-team-toolkit
+
+      - name: Add & Commit
+        uses: EndBug/add-and-commit@a94899bca583c204427a224a7af87c02f9b325d5
+        with:
+          add: '.'
+          github_token: ${{ inputs.appId != '' && steps.app-token.outputs.token || secrets.GH_TOKEN }}
+          message: 'feat: update OpenAPI schema'
+          push: 'false'
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@c5a7806660adbe173f04e3e038b0ccdcd758773c
```

Comment:
I'm not sure that we need this action, it is [quite simple](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/create-pr.yml)

Reply:
It also handles pushing the changes and branch creation. It's simpler to use this action rather than doing it manually.